doc/forge2d.md

```diff
@@ -0,0 +1,76 @@
+# Forge2D
+
+We (the Flame organization) maintains a ported version of the Box2D physics engine and our version is called Forge2D.
```

Comment:
I am not 100% sure, but I think here should `maintain`

Reply:
You can say both :)